guidelines/index.html

```diff
@@ -3755,8 +3755,14 @@ <h2 id="x6-glossary"><span class="secno">6. </span>用語集<span class="permali
    <p>共通の目的を共有し、同じコンテンツ制作者、グループ、又は組織により制作された<a href="#dfn-web-page-s" class="internalDFN" data-link-type="dfn">ウェブページ</a>の集合。
    </p>
    
-   <p class="example">例には複数のウェブページにまたがる発行物が含まれ、各ページには 一つの章又はその他の重要なセクションが含まれる。論理的に発行物は 一つの連続した単位であり、ウェブページ一式へのアクセスを可能にするナビゲーション機能を有する。
-   </p>
+   <aside class="example" id="example-26"><div class="marker">
+    <a class="self-link" href="#example-26">例<bdi> 26</bdi></a>
+  </div><p>以下の例がある:</p>
+      <ul>
+      <li>複数のウェブページにまたがる発行物で、各ページには一つの章又はその他の重要なセクションが含まれる。論理的に発行物は一つの連続した単位であり、ウェブページ一式へのアクセスを可能にするナビゲーション機能を有する。</li>
+      <li>e コマースのウェブサイトで、同じナビゲーション及び識別を共有するウェブページ一式に、商品が表示されている。しかし、精算プロセスに進むと、テンプレートが変更される。ナビゲーション及び他の要素が削除されるため、そのプロセスのページは機能的にも視覚的にも異なったものとなる。その精算ページは商品のページ一式の一部ではない。</li>
```

Comment:
個人の感覚としては、ここの so は「それによって」くらいのほうがしっくりきます。
単なる好みの問題ではありますが、どうでしょうか。

```suggestion
      <li>e コマースのウェブサイトで、同じナビゲーション及び識別を共有するウェブページ一式に、商品が表示されている。しかし、精算プロセスに進むと、テンプレートが変更される。ナビゲーション及び他の要素が削除され、それによって、そのプロセスのページは機能的にも視覚的にも異なったものとなる。その精算ページは商品のページ一式の一部ではない。</li>
```

Reply:
「so」の訳出、実は私自身、少し意訳がすぎて、しっくりこない (かと言ってうまい代案がない) 感じはありました。「それによって」はとてもよいと思います！